compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -157,6 +157,13 @@ featureExceptions = Feature
     , featureCppFlag = Just "DAML_EXCEPTIONS"
     }
 
+featureExperimental :: Feature
+featureExperimental = Feature
+    { featureName = "DAML Experimental"
+    , featureMinVersion = versionDev
+    , featureCppFlag = Just "DAML_EXPERIMENTAL"
```

Comment:
Do we need a cpp pragma here? If not, you can also set this to `Nothing`

Reply:
I the cpp pragma in `compiler/damlc/daml-stdlib-src/LibraryModules.daml` to be sure DA.Experimental module are visible only in 1.dev.